serialmfg/serial_resources/process_entry.py

```diff
@@ -119,7 +144,14 @@ def _upload_file(self, dataset_name, path, file_name, dataset_type):
         # if in the future we need to do a better guess, we can use python-magic to determine the mimetype
         files = {'file': (file_name, open(path, 'rb'), mimetypes.guess_type(path))} 
         storage_object = self.client.make_api_request("/files", "POST", files=files)
-        dataset = serial.Datasets.get(dataset_name, dataset_type, self.process_id)
+        dataset = None
+        try:
+            dataset = serial.Datasets.get(dataset_name, data_type, self.process_id)
```

Comment:
Should this be `dataset_type`? And should it default to file?

Reply:
Fixed, and why would it default to file?